mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -110,6 +111,64 @@ class TiltaksgjennomforingRepository(private val db: Database) {
         }
     }
 
+    fun updateEnheter(tiltaksnummer: String, enheter: List<String>): QueryResult<Int> = query {
+        @Language("PostgreSQL")
+        val findId = """
+            select id from tiltaksgjennomforing
+                where (:aar::text is null and split_part(tiltaksnummer, '#', 2) = :lopenr)
+                or (
+                    :aar::text is not null and split_part(tiltaksnummer, '#', 1) = :aar
+                    and split_part(tiltaksnummer, '#', 2) = :lopenr
+                )
+        """.trimIndent()
```

Comment:
Jeg regnet med det var via tiltaksnummeret man skulle finne ut hvilken gjennomføring det dreide seg om, siden det var sånn det var gjort i den sync job'en. Eller var det en annen måte?

Reply:
Alt du sier stemmer. I dev er mye av dataene som er lagt inn basert på et script som ble kjørt med faker.js en gang i fjor. Siden vi ikke har hatt noe kobling mellom Sanity-dokumenter og rader i databasen så er det mest sannsynlig lite til ingen match der.

I prod så stemmer det nok også godt at det er mange som ikke har tiltaksnummer i Sanity som matcher. I prod er det bare NAV Innlandet som har tatt i bruk Sanity, mens vi i db har alle tiltaksgjennomføringer for hele landet. Så alt høres riktig ut.